### PR TITLE
[don't merge] Remove Devon

### PR DESCRIPTION
**Pull request by tilda**
You know it :p

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,13 +17,6 @@ <h1 class="title"><strong>Staff</strong></h1>
       <h2 class="subtitle">Who's in charge?</h2>
 
       <!-- Management Team -->
-      <div class="columns">
```
Comment (ghost) [inline index.html]: Don't delete this line

**Review by ghost — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (tilda) [reply index.html]: Oh, oops.

Comment (tilda) [reply index.html]: Fixed `acb531e`

**Comment by tilda** [comment]
Putting on no-merge because of reworking stuffs by haden

**Comment by ghost** [comment]
#54

**Pull request closed without merging**